src/oc_erchef/apps/chef_objects/priv/depselector_rb/Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
 
-gem 'dep_selector', :git => 'git://github.com/opscode/dep-selector', :branch => 'master'
-gem 'erlectricity', :git => 'git://github.com/sdelano/erlectricity', :branch => 'master'
+gem 'dep_selector', :git => 'git://github.com/chef/dep-selector', :branch => 'master'
+gem 'erlectricity', :git => 'git://github.com/marcparadise/erlectricity', :branch => 'mp/gcc-4.9.2-compat-fix'
```

Comment:
Ugh, this should actually never have been committed here.  It was a temporary workaround for my local build.  Well - good to have it landed though, I'll repoint this one.